health_vara_acl.xml

```diff
@@ -7,33 +7,6 @@ contains the full copyright notices and license terms. -->
             <field name="name">Health Medical Assistant</field>
         </record>
 
-        <!-- Access to Menus -->
```

Comment:
These menus are gone. We might have to run this by Hannes as it will likely mean some retraining.